open_humans/templatetags/utilities.py

```diff
@@ -312,7 +312,8 @@ def render_user_badges(member, badge_class='mini-badge'):
     Returns the html to render all of a member's badges.
     """
     projects = DataRequestProjectMember.objects.select_related('project').filter(
-                   visible=True, project__approved=True).filter(member_id=member.id)
+                   visible=True, project__approved=True, member_id=member.id,
```

Comment:
is `member_id=member.id` the same as `member=member`?

Reply:
Oops, wasn't paying attention.